_distro_map.yml

```diff
@@ -9,6 +9,9 @@ openshift-origin:
     master:
       name: Latest
       dir: latest
+    enterprise-3.2:
+      name: '1.2'
+      dir: 1.2
```

Comment:
I tried cherry-picking this to my local master and then building (`asciibinder build -a`) but it fails with:

```
Building OpenShift Origin for branch 'enterprise-3.2'
Error: Could not generate docs:
TypeError: no implicit conversion of Float into String at
    /home/durant/.gem/ruby/gems/ascii_binder-0.1.5/lib/ascii_binder/helpers.rb:610:in `join'
    /home/durant/.gem/ruby/gems/ascii_binder-0.1.5/lib/ascii_binder/helpers.rb:610:in `block (2 levels) in generate_docs'
    /home/durant/.gem/ruby/gems/ascii_binder-0.1.5/lib/ascii_binder/helpers.rb:575:in `each'
    /home/durant/.gem/ruby/gems/ascii_binder-0.1.5/lib/ascii_binder/helpers.rb:575:in `block in generate_docs'
    /home/durant/.gem/ruby/gems/ascii_binder-0.1.5/lib/ascii_binder/helpers.rb:539:in `each'
    /home/durant/.gem/ruby/gems/ascii_binder-0.1.5/lib/ascii_binder/helpers.rb:539:in `generate_docs'
    /home/durant/.gem/ruby/gems/ascii_binder-0.1.5/bin/asciibinder:15:in `call_generate'
    /home/durant/.gem/ruby/gems/ascii_binder-0.1.5/bin/asciibinder:291:in `<top (required)>'
    /home/durant/bin/asciibinder:23:in `load'
    /home/durant/bin/asciibinder:23:in `<main>'.
```

However, it works if I changed it to something like:

```
    enterprise-3.2:
      name: '1.2'
      dir: origin/1.2
```